dspace/config/dspace.cfg

```diff
@@ -906,6 +906,15 @@ org.dspace.app.itemexport.life.span.hours = 48
 # cumulative sizes are more than this entry the export is not kicked off
 org.dspace.app.itemexport.max.size = 200
 
+### Bulkedit Metadata export settings
+# The maximum amount of items that can be exported using the "metadata-export" script / process
+# Recommend to keep this at a feasible number, as exporting large amounts of items can be resource intensive
+metadataexport.max.items = 1
```

Comment:
This looks like accidental test configs?  I'm assuming this should be commented out and set to 500 by default (not 1).

Please also add a note to the comment above that the default is 500.

Reply:
Indeed, that shouldn't of have came through 🙈, I have fixed it now.

Currently Administrators also have this restriction applied by default (can be modified with the property) but not sure if we want this to be enabled / disabled by default